assignments/callbacks.js

```diff
@@ -24,25 +24,67 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr);
 }
 
+function arrlength(arr) {
+  return arr.length
+}
+
+console.log(getLength(items, arrlength));
+
+
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr)
 }
 
+function lastItem(arr) {
+  return arr[arr.length-1]
+}
+
+console.log(last(items,lastItem));
+
+
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x,y);
+}
+
+function sum(x,y){
+  return x+y;
 }
 
+console.log(sumNums(3,4, sum));
+
+
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x,y);
+}
+
+function multiply(x,y) {
+  return x * y;
 }
+console.log(multiplyNums(3, 4, multiply));
+
+
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  return cb(item,list)
```

Comment:
read carefully, this tells you that you should pass true or false to cb, not item or list.